MonoGame.Framework/Graphics/Effect/IEffectLights.cs

```diff
@@ -73,9 +35,8 @@ public interface IEffectLights
 		bool LightingEnabled { get; set; }
 
         /// <summary>
-        /// Initializes the lights to the standard key/fill/back lighting rig.
+        /// Initializes the default lights.

```

Comment:
I'm ok with making the interface doc a little more general...  it is possible some new `Effect` overloads this with some other default lighting rig.

However...  the information there is 100% accurate as to what the function does with the MonoGame stock effects (see http://blogs.msdn.com/b/shawnhar/archive/2007/04/09/the-standard-lighting-rig.aspx).

If you want to remove the information here then you need to add it back here:

https://github.com/mono/MonoGame/blob/develop/MonoGame.Framework/Graphics/Effect/BasicEffect.cs#L406

Since the inherited doc from the interface will now be less accurate.